napari/layers/labels/labels.py

```diff
@@ -878,6 +879,10 @@ def _get_color_lookup_func(self, data, min_label_val, max_label_val):
             self._all_vals[0] = 0
         return self._lookup_with_index
 
+    def _update_slice_response(self, response: _ImageSliceResponse) -> None:
+        response = response.to_displayed(self._raw_to_displayed)
```

Comment:
Unsure about this change. I like it because it isolates the complication of `_raw_to_displayed` to `Labels`, which is the only place it matters. It also means we don't need to pass around a method bound to a weak reference to the layer.

I don't like it because it relies on inheritance and an override.

Reply:
I'm confused why this change is being added here. I know Labels are... different... from the other types. It seems to me you aren't making them an `AsyncSlicable` layer since `_make_slice_request` is not included. So why are you including `_update_slice_response` here?

Reply:
Labels inherit from `_ImageBase` which does define `_make_slice_request`.